src/utils/nlpUtils/similarityCheck.py

```diff
@@ -0,0 +1,75 @@
+'''
+Created on Sep 20, 2022
+
+@author: mandd
+'''
```

Comment:
Copyright statement need to be added when merge into devel

Reply:
added